tests/testthat/test-expressions.R

```diff
@@ -327,4 +327,10 @@ with_test_authentication({
         skip("'which' isn't implemented correctly")
         expect_identical(which(duplicated(ds$v4)), 3:20)
     })
+
+    test_that("rollupResolution can be set", {
+        expect_null(rollupResolution(ds$v5))
+        expect_silent(rollupResolution(ds$v5) <- "M")
```

Comment:
Thanks for adding these tests. For consistency, I would remove this `expect_silent()`, since any warnings or errors will propagate and fail the build regardless. This isn't my first tendency either, I usually do `expect_silent` explicitly, but we've eschewed it many (all?) other places in the package.